images/metalk8s-keepalived/liveness-probe.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+/generate-config.py --input /etc/keepalived/keepalived-input.yaml --template /etc/keepalived/keepalived.conf.j2 --output /etc/keepalived/keepalived.conf.liveness && diff -w /etc/keepalived/keepalived.conf /etc/keepalived/keepalived.conf.liveness
```

Comment:
Why `diff -w` ? I would go with a `-q` instead no ? And to me there is no need to actually create the configuration file just give it as input to diff

If we could also have a simple comment about this check

I would change it like that 

```suggestion
#!/bin/bash

set -xue -o pipefail

# Simple check to ensure that the config does not need to be updated
/generate-config.py --input /etc/keepalived/keepalived-input.yaml --template /etc/keepalived/keepalived.conf.j2 | diff -q /etc/keepalived/keepalived.conf -
```

Reply:
`diff -w` is for not considering the changes about whitespaces which I had when regenerating the config file

Reply:
Oh indeed, I read the beginning of diff helper and ....

```console
$ LANG= diff --help | grep "\-w" 
  -W, --width=NUM               output at most NUM (default 130) print columns
  -w, --ignore-all-space          ignore all white space
$ LANG=fr_FR.utf8 diff --help | grep "\-w"
  -w, --width=N                Limiter la sortie à au plus N colonnes imprimées (130 par défaut)
  -w, --ignore-all-space        Ignorer tout blanc d'espacement
```
NOTE: The double `-w` lowercase in FR ... 

Ok let's put both